### PR TITLE
Support write operators in aggregate store

### PR DESCRIPTION
**Pull request by howardjohn**
Will be useful for https://github.com/istio/istio/pull/25434

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

File: pilot/pkg/config/aggregate/config.go [hunk 1]
```diff
@@ -29,9 +29,9 @@ import (
 
 var errorUnsupported = errors.New("unsupported operation: the config aggregator is read-only")
 
-// Make creates an aggregate config store from several config stores and
+// make creates an aggregate config store from several config stores and
```
Comment (ramaraochavali) [inline pilot/pkg/config/aggregate/config.go:32]: nit: /s/make/makeStore

File: pilot/pkg/config/aggregate/config_test.go [hunk 2]
```diff
@@ -113,22 +112,76 @@ func TestAggregateStoreList(t *testing.T) {
...
+		},
+	}); err != nil {
+		t.Fatal(err)
+	}
+
+	l, err := store.List(collections.K8SServiceApisV1Alpha1Httproutes.Resource().GroupVersionKind(), "")
+	g.Expect(err).NotTo(gomega.HaveOccurred())
+	g.Expect(l).To(gomega.HaveLen(1))
+
+	l, err = store1.List(collections.K8SServiceApisV1Alpha1Httproutes.Resource().GroupVersionKind(), "")
```
Comment (ramaraochavali) [inline pilot/pkg/config/aggregate/config_test.go:147]: Can you compare this `l `and above `l` and make sure they are same?

Comment (howardjohn) [reply pilot/pkg/config/aggregate/config_test.go]: yes, done

**Review by ramaraochavali — APPROVED** [review]
Thank you

**Pull request merged**